msteams-platform/tabs/how-to/using-teams-client-library.md

```diff
@@ -59,7 +59,7 @@ Enabling an existing Teams app to run in Outlook and Microsoft 365 requires all
 
 2. [Modifying existing application code](#2-update-teamsjs-references) according to the required changes described in this article, and
 
-3. [Updating your app manifest](#3-update-the-manifest-optional) to version 1.13 or later.
+3. [Updating your app manifest(previously called Teams app manifest)](#3-update-the-app-manifest-optional) to version 1.13 or later.
```

Comment:
#3-update-the-app-manifest-optional
Please verify if this section is also renamed.

Reply:
Yes, its renamed